integration-tests-ex/cases/cluster/S3DeepStorage/docker-compose.yaml

```diff
@@ -0,0 +1,121 @@
+# limitations under the License.
+
+networks:
+  druid-it-net:
+    name: druid-it-net
+    ipam:
+      config:
+        - subnet: 172.172.172.0/24
+
+services:
+  zookeeper:
+    extends:
+      file: ../Common/dependencies.yaml
+      service: zookeeper
+
+  metadata:
+    extends:
+      file: ../Common/dependencies.yaml
+      service: metadata
+
+  coordinator:
+    extends:
+      file: ../Common/druid.yaml
+      service: coordinator
+    container_name: coordinator
+    environment:
+      - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
+      # The frequency with which the coordinator polls the database
+      # for changes. The DB population code has to wait at least this
+      # long for the coordinator to notice changes.
+      - druid_manager_segments_pollDuration=PT5S
+      - druid_coordinator_period=PT10S
+      - AWS_REGION=${AWS_REGION}
+      - druid_s3_accessKey=${AWS_ACCESS_KEY_ID}
+      - druid_s3_secretKey=${AWS_SECRET_ACCESS_KEY}
+    depends_on:
+      - zookeeper
+      - metadata
+
+  overlord:
+    extends:
+      file: ../Common/druid.yaml
+      service: overlord
+    container_name: overlord
+    environment:
+      - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
+      - AWS_REGION=${AWS_REGION}
+      - druid_s3_accessKey=${AWS_ACCESS_KEY_ID}
+      - druid_s3_secretKey=${AWS_SECRET_ACCESS_KEY}
+    depends_on:
+      - zookeeper
+      - metadata
+
+  broker:
+    extends:
+      file: ../Common/druid.yaml
+      service: broker
+    environment:
+      - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
+      - AWS_REGION=${AWS_REGION}
+      - druid_s3_accessKey=${AWS_ACCESS_KEY_ID}
+      - druid_s3_secretKey=${AWS_SECRET_ACCESS_KEY}
+    depends_on:
+      - zookeeper
+
+  router:
+    extends:
+      file: ../Common/druid.yaml
+      service: router
+    environment:
+      - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
+      - AWS_REGION=${AWS_REGION}
+      - druid_s3_accessKey=${AWS_ACCESS_KEY_ID}
+      - druid_s3_secretKey=${AWS_SECRET_ACCESS_KEY}
+    depends_on:
+      - zookeeper
+
+  historical:
+    extends:
+      file: ../Common/druid.yaml
+      service: historical
+    environment:
+      - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
+      - AWS_REGION=${AWS_REGION}
+      - druid_s3_accessKey=${AWS_ACCESS_KEY_ID}
+      - druid_s3_secretKey=${AWS_SECRET_ACCESS_KEY}
+    depends_on:
+      - zookeeper
+
+  indexer:
+    extends:
+      file: ../Common/druid.yaml
+      service: indexer
+    environment:
+      - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
+      - druid_test_loadList=druid-s3-extensions
+      - druid_storage_type=s3
+      - druid_storage_bucket=${DRUID_CLOUD_BUCKET}
+      # Using DRUID_CLOUD_PATH env as baseKey as well.
+      - druid_storage_baseKey=${DRUID_CLOUD_PATH}
+      - druid_s3_accessKey=${AWS_ACCESS_KEY_ID}
+      - druid_s3_secretKey=${AWS_SECRET_ACCESS_KEY}
```

Comment:
Let's document the required env vars somewhere. Perhaps we add a Markdown page that lists the requirements for each test. Or, we document these at the top of the Java file. Be sure to list the env vars used by the server vs. those used by the test.

Reply:
It is already documented in this file - https://github.com/apache/druid/blob/master/integration-tests-ex/docs/test-config.md